moto/awslambda_simple/models.py

```diff
@@ -20,9 +24,10 @@ def invoke(
         response_headers: Any,
     ) -> Optional[Union[str, bytes]]:
 
-        if body:
-            return str.encode(body)
-        return b"Simple Lambda happy path OK"
+        default_result = ""
```

Comment:
```suggestion
        default_result = str.encode(body) if body else b"Simple Lambda happy path OK"
```

No reason to change that for existing  users - I'm sure someone's tests depend on that specific output :slightly_smiling_face:

Reply:
I agree that an empty string makes a lot more sense, btw - but it is technically a breaking change, so we should only do that as part of a major release.

Reply:
Good point but the rationale is: this more accurately reflects what happens with real lambda in AWS, by default nothing is returned.

Happy to change this back though 🤷